translate/storage/php.py

```diff
@@ -54,13 +54,133 @@
 `String type <http://www.php.net/language.types.string>`_.
 """
 
-import logging
 import re
 import six
+from phply.phpparse import make_parser
+from phply.phplex import lexer, FilteredLexer, full_lexer, unparsed
+from phply.phpast import InlineHTML, FunctionCall, Assignment, ArrayOffset, Variable, Array, ArrayElement, Return, BinaryOp, Node
```

Comment:
These imports don't follow our import guidelines: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports

Also the lines shouldn't be that long.

Reply:
wierd that it didnt break any linting checks @unho